Add option to ignore all virtual members

Cyaminthe opened the pull request:
Similarly to `IncludeCompilerGeneratedMembers`, adds optional metadata `IncludeVirtualMembers`.
When it is set to false, all virtual members in the assembly are exempt from publicizing (unless specified explicitly).
The default value is true, to not alter previously expected behaviour.

krafs commented on README.md:
````diff
@@ -94,34 +94,38 @@ This compiles just fine. However, during execution the runtime is presumably loa
 </ItemGroup>
 ```
 
+However, if there is a lot of protected members you have to override, doing this for all of them can be cumbersome.
````
typo: if there _are_ a lot of...

krafs commented on src/Publicizer/Publicizer.csproj:
```diff
@@ -27,7 +27,7 @@
     <PackageIcon>icon.png</PackageIcon>
     <PackageReadmeFile>README.md</PackageReadmeFile>
-    <Version>2.0.1</Version>
+    <Version>2.1.0</Version>
```
I appreciate you trying to do as much work as possible in this PR to make it ready for a release, but I think bumping the version should be done separate from the new feature. Please revert, and I'll bump it before release.

krafs reviewed (changes requested):
Thanks for the PR! Contributors are always welcome :)

Besides a couple of minor comments, I'd still like some kind of test for this. I realize that the existing tests are bulky and hard to read, but they are better than nothing. If you can add a similar test for IncludeVirtualMembers (e.g. adapting the test for CompilerGeneratedMembers) I'd feel better approving the PR.

Again - thanks for helping out!

Cyaminthe commented:
> Besides a couple of minor comments, I'd still like some kind of test for this. I realize that the existing tests are bulky and hard to read, but they are better than nothing. If you can add a similar test for IncludeVirtualMembers (e.g. adapting the test for CompilerGeneratedMembers) I'd feel better approving the PR.

Sure thing. Added a test (and fixed the minor comments).

krafs commented:
Thanks! I'll try to release soon.
